### PR TITLE
[test] Fix containers size for screenshots

### PR DESCRIPTION
**Pull request by oliviertassinari**
Implement https://github.com/mui-org/material-ui-x/pull/1081#issuecomment-784337549

### CODE REVIEW

File: test/regressions/TestViewer.js [hunk 1]
```diff
@@ -27,10 +28,27 @@ const styles = (theme) => ({
     display: 'inline-block',
     padding: theme.spacing(1),
   },
+  dataGridContainer: {
+    minHeight: 400,
+    maxWidth: 500,
+    display: 'flex',
+    flexDirection: 'column',
+    // Workaround the min-height limitation
```
Comment (oliviertassinari) [inline test/regressions/TestViewer.js:36]: https://codepen.io/oliviertassinari/pen/dyOJNqm

**Comment by oliviertassinari** [comment]
I have approved the changes, it seems correct:

Screenshot 2021-02-24 at 22 01 05

**Comment by oliviertassinari** [comment]
The visual regressions are a bit slower now that they are all larger. But it still seems manageable :).

@eps1lon an idea I got working on this and seeing console warnings, errors in some of the stories. Maybe we could mock `console.error`/`console.warn` and fail the test case if something pop? It could be a quick-win a way to assert that all the demos we have in the documentation are "clean" (warning free)

File: test/regressions/TestViewer.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
+import clsx from 'clsx';
```
Comment (dtassone) [inline test/regressions/TestViewer.js:3]: why not use the `classnames` utils like everywhere else, so we avoid the dependency?

**Review by dtassone — APPROVED** [review]

**Comment by eps1lon** [comment]
You can [listen to console calls in playwright](https://playwright.dev/docs/api/class-page#pageonconsole) and then fail the test with warn/error calls.. Mocking is always a bit brittle since you have to take care to properly restore them.

**Pull request merged**